app/invocation/cache_requests_card.tsx

```diff
@@ -477,6 +502,16 @@ function cacheTypeTitle(cacheType: cache.CacheType): string | undefined {
   }
 }
 
+function renderCompressionSavings(result: cache.ScoreCard.IResult) {
+  const compressionSavings = 1 - Number(result.transferredSizeBytes) / Number(result.digest.sizeBytes);
+  return (
+    <span className={`compression-savings ${compressionSavings > 0 ? "positive" : "negative"}`}>
+      {compressionSavings <= 0 ? "+" : ""}
+      {(-compressionSavings * 100).toPrecision(3)}%
```

Comment:
Thoughts on just showing percentage here? (You could hover if you wanted to see the new size).

Reply:
I was on the fence, but it's probably cleaner to show just the percentage. Done :+1: